### DISCUSSION TITLE
Simplify the command to run and install a pipeline

### DISCUSSION THREAD

**Discussion by yorkie** [alibaba/pipcook#669] [Ideas]
To install a pipeline, we have to use the complete URL:

```
pipcook pipeline install https://raw.githubusercontent.com/alibaba/pipcook/main/example/pipelines/text-bayes-classification.json
```

This looks too long, how about adding a new protocol `github://` to simplify it:

```
pipcook pipeline install github://alibaba@pipcook/example/pipelines/text-bayes-classification.json
```

**Comment by yorkie** [comment]
Another alternative:

```
pipcook pipeline install --github alibaba/example:pipelines/text-bayes-classification
```

**Comment by FeelyChau** [comment]
I prefer to paste the url, so it's useless for me :)

**Reply by yorkie** [reply]
> I prefer to paste the url, so it's useless for me :)

Ok :p

**Comment by SirM2z** [comment]
URL is more general and easier to understand

**Comment by rickycao-qy** [comment]
i think it's a little bit complicated to add another protocol. I suppose the URL is long but easier to understand
